root/about.html

```diff
@@ -18,12 +18,12 @@
 
 MetaCPAN has two parts:
 
-* [metacpan.org](https://metacpan.org), a front end to...
-* [api-v1.metacpan.org](http://api-v1.metacpan.org/) the API
+* [<% mcpan_web %>](<% web_secure %>), a front end to...
+* [<% mcpan_api %>](<% api_secure %>) the API
 
 So
-[https://metacpan.org/module/Moose](https://metacpan.org/module/Moose)
-vs [http://api-v1.metacpan.org/module/Moose](http://api-v1.metacpan.org/module/Moose)
+[<% web_secure %>/module/Moose](<% web_secure %>/module/Moose)
```

Comment:
The canonical urls in the templates SHOULD be hard coded - but thats just for SEO

> On 10 Jun 2016, at 16:44, Olaf Alders <notifications@github.com> wrote:
> 
> In root/about.html:
> 
> >  
> >  So
> > -[https://metacpan.org/module/Moose](https://metacpan.org/module/Moose)
> > -vs [http://api-v1.metacpan.org/module/Moose](http://api-v1.metacpan.org/module/Moose)
> > +[<% web_secure %>/module/Moose](<% web_secure %>/module/Moose)
> I think we should use relative URLs wherever possible in the templates. We could probably remove the variable from a lot of these.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>